src/main/java/asg/asts/Generator.java

```diff
@@ -1169,12 +1169,6 @@ private void generateList_interface(ListDef l) {
 
         createCopyWithRefsMethod(l, sb);
 
-        // deprecate generic set
```

Comment:
Is this used somewhere? I deleted it, it caused deprecation warnings and is not used anywhere as far as I could see